WebRoot/js/zimbraMail/mail/view/ZmComposeView.js

```diff
@@ -1439,9 +1439,9 @@ ZmComposeView.prototype.enableInputs =
 function(bEnable) {
     DBG.println('draft', 'ZmComposeView.enableInputs for ' + this._view + ': ' + bEnable);
     this._recipients.enableInputs(bEnable);
-    if (typeof(this._subjectField) !== 'undefined') {
```

Comment:
not exactly sure, we should be doing try/catch
can you check if below fix helps?
`if (typeof(this._subjectField) !== 'undefined' && typeof(this._bodyField) !== 'undefined') {`